app/domain/logs.rb

```diff
@@ -277,65 +277,65 @@ module AuthnJwt
 
       IssuerResourceNameConfiguration = ::Util::TrackableLogMessageClass.new(
         msg: "\"issuer\" value will be taken from '{0-resource-id}'",
-        code: "CONJ00054D"
+        code: "CONJ00054I"
       )
 
       RetrievedIssuerValue = ::Util::TrackableLogMessageClass.new(
         msg: "Retrieved \"issuer\" with value '{0}'",
-        code: "CONJ00055D"
+        code: "CONJ00055I"
       )
 
       ParsingIssuerFromUri = ::Util::TrackableLogMessageClass.new(
         msg: "Parsing \"issuer\" value from '{0}'...",
         code: "CONJ00056D"
       )
 
-      JWTAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
-        msg: "Started JWT authentication flow for vendor '{0}'",
-        code: "CONJ00057D"
+      JwtAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
+        msg: "Started JWT authentication flow for '{0-authenticator-name}' Authenticator",
+        code: "CONJ00057I"
       )
 
-      URL_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "The JWT ID is included in and will be retrieved from the URL",
+      SelectingIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selecting identity provider interface...",
         code: "CONJ00058D"
       )
 
-      DECODED_TOKEN_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "JWT Identity in decoded token is available and will be retrieved from it",
-        code: "CONJ00059D"
+      SelectedIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selected identity provider interface: '{0-identity-provider-interface-name}'",
+        code: "CONJ00059I"
       )
 
-      LOOKING_FOR_IDENTITY_FIELD_NAME = ::Util::TrackableLogMessageClass.new(
-        msg: "Looking for variable field name in '{0}'...",
-        code: "CONJ00060D"
+      RetrievedResourceValue = ::Util::TrackableLogMessageClass.new(
+        msg: "Retrieved value '{0-resource-value}' of resource name '{1-resource-name}'",
```

Comment:
```suggestion
        msg: "Retrieved '{0-resource-value}' value from '{1-resource-name}' resource",
```

Reply:
@nessiLahav  please provide example for these kinds of errors so that I can understand how it will look when fill in. This will help me understand how to place the words (adjectives/nouns) in the sentence.